content-elements/layout/col-one/index.js

```diff
@@ -33,4 +33,5 @@ element.withElementId('col-one-l2ZclN')
         require('../../base/banner'),
         ...contentElements,
         ...advancedElements,
-        require('../../form/form-container')));
\ No newline at end of file
+        require('../../form/form-container'),
+        require('../../form/form-pin')));
```

Comment:
Das Pin Element muss in einem Formular sein - es darf also nicht hier rein

Reply:
Ich habe im letzten Commit das Formular mit dem Pin Element zu einem kombined. Deshalb gibt es das schon irgendwie noch, nur, dass es jetzt ein Teil vom Pin Element ist.

Reply:
Weil sich das Pin Element schon automatisch in einem Formular befindet. Müsste das eigentlich so passen?